include/seqan3/alignment/pairwise/alignment_configurator.hpp

```diff
@@ -257,14 +335,14 @@ constexpr function_wrapper_t alignment_configurator::configure_free_ends_initial
     auto _leading_both = [&](auto first_seq, auto second_seq) constexpr
     {
         // Define the trait for the initialisation policy
-        struct _trait
+        struct _trait_type
```

Comment:
They have fun?!

Reply:
![](https://media.makeameme.org/created/are-you-5acbb2.jpg)

Reply:
🤣

Reply:
you really want me to change this?

Reply:
well we don't use `_` anywhere except to mark "private" variables in aggregate types or in the case of name clashes of local variables. I don't see the necessity here to use it and it makes the names stand out needlessly..

Reply:
For the lambda it is true. They are not seen outside of the scope, but with the traits type I wanted to avoid possible name clashes with function or class templates

Reply:
And last time I complained about your `_char_type` thing you were like: "Na, this is just more readable." 
...
just saying 😉

Reply:
Yeah, well I had two things there were 100% identical in all situations were code is generated so I thought it was important to give them very similar names. In this case I see no obvious reason for the underscore, that's why I am asking. Stuff is private anyway, it doesn't need to be private-private does it?

Reply:
I changed it already